### PR TITLE
Improvement: Add option to always use a file's dir as current working…

### PR DESCRIPTION
**Pull request by mwick83**
… dir

The added option lets the user configure to always use a file's
directory as the current working directory - instead of the internal
heuristic. This will only be done if explicitly configured and the
uri.scheme is 'file'.

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -176,6 +176,13 @@ function getCurrentWorkingDirectory(): string {
 
     const isFileOrUntitledDocument = uri && (uri.scheme === 'file' || uri.scheme === 'untitled');
     if (isFileOrUntitledDocument) {
+        const useDocumentDirAsWorkDir = vscode.workspace.getConfiguration('filterText').useDocumentDirAsCwd;
```
Comment (yhirose) [inline src/extension.ts]: @mwick83, `vscode.workspace.getConfiguration('filterText').useDocumentDirAsCwd` should be `vscode.workspace.getConfiguration('filterText').useDocumentDirAsWorkDir`?

**Comment by mwick83** [comment]
Yes, you are right. I did a last minute change of the name and forgot the essential part ;-)

File: src/extension.ts (hunk 1 above)
Comment (mwick83) [reply src/extension.ts]: Yes, you are right. I did a last minute change of the name and forgot the essential part ;-)

**Comment by yhirose** [comment]
@mwick83, thanks for your contribution! I also updated the version to 0.0.12.

**Pull request merged**